pkg/daemon/discover/discover.go

```diff
@@ -167,9 +172,27 @@ func udevBlockMonitor(c chan string, period time.Duration) {
 	// return any add or remove events, but none that match device mapper
 	// events. string matching is case-insensitve
 	events := make(chan string)
+
+	//  variable to store the regular expression
+	var udevregex []string
+
+	// get discoverDaemonUdevBlacklist from the enviornment variable
+	// if 'nil' generate the regex; else use the regex provided by user
+	discoverUdev := os.Getenv(discoverDaemonUdev)
+	if discoverUdev == "nil" {
+		for _, re := range udevFilter {
+			udevregex = append(udevregex, (regexp.MustCompile("(?i)" + re + "[0-9]+")).String())
```

Comment:
`discoverUdev` variable will have whatever value the user is inputting right? So the user can input their own customized regex values. The append works only if the user didn't provide any values. I will comment the default used regex in the `operator.yaml` so that a user will know what it is we are using as the default. If a user wishes to make changes on the regex, then he can do so in the env variable itself.